test/puppetlabs/trapperkeeper/services/watcher/filesystem_watch_service_test.clj

```diff
@@ -385,3 +385,22 @@
        (is (logged?
             #"shutdown-on-error triggered because of exception"
             :error))))))
+
+(deftest ^:integration overflow-test
+  (let  [watch-dir  (fs/temp-dir "overflow")
+        results  (atom  [])
+        callback  (fn  [events]  (swap! results concat events))]
```

Comment:
This call to `swap!` can be replaced with `(make-callback results)` which is consistent with the other tests.